common/lib/capa/capa/tests/test_responsetypes.py

```diff
@@ -2173,7 +2175,6 @@ def test_valid_xml(self):
         Test that `build_problem` builds valid xml
```

Comment:
I think you should just remove this test entirely. `build_problem` is called in `_make_problem`, which other tests call in this class. A failure here will arise in other tests. This test is unnecessary.